### ISSUE TITLE
Docs: Multiple (inconsistent) lists of PSF Code of Conduct Work Group members

### ISSUE THREAD

**Issue by JesseWeinstein** [proevo/pythondotorg#2679]
### Summary

The current membership of the PSF CoC WG is listed in two different places: 

https://www.python.org/psf/workgroups/ (along with the other workgroups)

and 

https://wiki.python.org/psf/ConductWG/Charter#List_of_Participants.2FWho_we_are

The all-workgroups one is currently (slightly) out of date (it doesn't include Jeremy Hylton, who was added to the wiki one on Dec 12, 2024), and (like most of the rest of the content of python.org) it lacks a public commit history, so I'd suggest that it be changed to refer to the wiki page, instead of repeating the list there.

**Comment by hugovk** [comment]
Yes, we could edit https://www.python.org/psf/workgroups/ to say something like:

> The list of members can be found [here](https://www.python.org/psf/workgroups/).

cc WG co-chair @sleepypioneer, how does that sound? I can make the edit.

**Comment by sleepypioneer** [comment]
Makes alot of sense to me, we also struggle with consistency across spaces. I'll raise this issue in our meeting today but I think the proposal is good.

**Comment by sleepypioneer** [comment]
Following up after our meeting, we would very happy if you can make that edit @hugovk thank you so much! And thank you @JesseWeinstein for reporting 🙌

**Comment by hugovk** [comment]
Done, thanks all!

https://www.python.org/psf/workgroups/#code-of-conduct-work-group

**Comment by JesseWeinstein** [comment]
Unfortunately, the link that was updated doesn't point to the wiki, but back to the same page the link is on! It should point to https://wiki.python.org/psf/ConductWG/Charter#List_of_Participants.2FWho_we_are

I do appreciate your attending to this, though!

**Comment by hugovk** [comment]
Oops! Thanks for checking, fixed now!
